puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/test_kafo_pull_request.yaml

```diff
@@ -0,0 +1,50 @@
+- job:
+    name: test_kafo_master_pull_request
+    project-type: matrix
+    logrotate:
+      daysToKeep: 45
+      numToKeep: -1
+    properties:
+      - github:
+          url: https://github.com/theforeman/kafo
+    wrappers:
+      - abort_build:
+          timeout_min: 60
+    parameters:
+      - string:
+          name: branch
+          default: master
+          description: "Branch to run test suite on"
+      - pr_parameters
+    scm:
+      - kafo_pr
+    triggers:
+      - github_pr:
+          context: 'kafo'
+    builders:
+      - test_kafo
+    axes:
+      - axis:
+          type: user-defined
+          name: ruby
+          values:
+          - 2.0.0
+          - 2.1
+          - 2.2
+          - 2.3
+          - 2.4
+      - axis:
+          type: user-defined
+          name: puppet
+          values:
+          - 4.10.0
```

Comment:
Should this be latest of 4.10.Z line?

Reply:
I think that this should be true since we use ```~> VERSION```. Maybe it could also be ```4.0``` or ```4.10``` since that should also match it. See https://github.com/theforeman/kafo/blob/93aaa70eff832fc6ccfbcbbf672f5cd775007420/Gemfile#L22